src/gc-heap-snapshot.cpp

```diff
@@ -401,8 +401,7 @@ void _gc_heap_snapshot_record_hidden_edge(jl_value_t *from, void* to, size_t byt
 }
 
 static inline void _record_gc_edge(const char *node_type, const char *edge_type,
-                                   jl_value_t *a, jl_value_t *b, size_t name_or_idx) JL_NOTSAFEPOINT
-{
+                                   jl_value_t *a, jl_value_t *b, size_t name_or_idx) JL_NOTSAFEPOINT {
```

Comment:
style goes the other way here (no particular reason, it just does)

Reply:
Alrighty. Switched it back.

Reply:
you need to do all the others in this PR too

otherwise, this looks good to merge to me